### PR TITLE
Fields for classes

### PR DESCRIPTION
**Pull request by taylorhansen**
I also added a `protected` access modifier.

### I have:
 - [x] Ran a test (`npm run dev && npm test`)
 - [x] Added tests for my changes
 - [x] Documented my code thoroughly.

### CODE REVIEW

**Comment by vihanb** [comment]
> Merging #28 into develop will decrease coverage by 25.55%

is this up to date with develop?

**Comment by taylorhansen** [comment]
@vihanb I dont think it _is_ up to date with develop. :P I'll merge it and handle the conflicts

File: test/Parser/classes.js [hunk 1]
```diff
@@ -17,6 +17,25 @@ export default () => describe("Classes", () => {
             class A {}`
     });
 
+    describe('Fields', () => {
+        valid`class A { public let goat }`;
+        valid`class A { public let goat = 1337 }`;
```
Comment (vihanb) [inline test/Parser/classes.js:22]: 12/10 quality test

Comment (taylorhansen) [reply test/Parser/classes.js:22]: do u even `13375p34k`?

**Pull request merged**